docs_src/microservices/security/Dockerfile-remote-sshd

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:16.04
```

Comment:
Can this be done with an alpine or busybox base image instead?

Reply:
i think it can. this is just an example. does not have to be from ubuntu